src/EditorFeatures/TestUtilities/Classification/AbstractClassifierTests.cs

```diff
@@ -64,8 +64,8 @@ protected async Task TestAsync(
 
             var actualOrdered = actual.OrderBy((t1, t2) => t1.TextSpan.Start - t2.TextSpan.Start);
 
-            var actualFormatted = actualOrdered.Select(a => new FormattedClassification(allCode.Substring(a.TextSpan.Start, a.TextSpan.Length), a.ClassificationType));
-            AssertEx.Equal(expected, actualFormatted);
+            var actualFormatted = actualOrdered.SelectAsArray(a => new FormattedClassification(allCode.Substring(a.TextSpan.Start, a.TextSpan.Length), a.ClassificationType));
+            AssertEx.Equal(expected.ToImmutableArray(), actualFormatted);
```

Comment:
this improves the diff view as the immutablearray overload produces a better diff.